checker/tests/nullness/Issue500.java

```diff
@@ -0,0 +1,16 @@
+import java.util.AbstractList;
```

Comment:
By convention, we have test cases for issues start with:
// Test case for Issue 419:
// https://github.com/typetools/checker-framework/issues/419